### PR TITLE
fix tooltip cutoff, remove linkability of thumbnail

### PR DESCRIPTION
**Pull request by eonwarped**
The link doesn't really behave as expected (esp when using multi-equip).

### CODE REVIEW

**Comment by eonwarped** [comment]
it's not perfect because the arrow part of the tooltip doesn't align anymore, but that is proving to be a lot trickier. maybe i can style to remove it entirely though

**Comment by pyc92** [comment]
> The link doesn't really behave as expected (esp when using multi-equip).
I don't really understand what you meant. Could you please explain this more or provide a screenshot?

**Comment by eonwarped** [comment]
> > The link doesn't really behave as expected (esp when using multi-equip).
> 
> I don't really understand what you meant. Could you please explain this more or provide a screenshot?

Screenshot 2023-04-06 at 10 47 13 PM
Screenshot 2023-04-06 at 10 47 24 PM

Second image shows what happens when clicking, the url changes but we are still selecting for the previous otter. So maybe I can disable it when it's used in this modal but keep link in my-items view.

**Comment by eonwarped** [comment]
Added style to remove the arrow part, and restored link for my items view.

**Comment by pyc92** [comment]
Okay, I understand. However, why do we need to overwrite the tooltip position? I think react-tooltip already handles it quite well.

**Comment by eonwarped** [comment]
Here's the before and after:

Screenshot 2023-04-06 at 11 11 40 PM
Screenshot 2023-04-06 at 11 12 12 PM

I think it's specifically needed for this view because of the small frame, my items is probably fine. I removed the arrow because otherwise it doesn't point to the bubble when it gets moved, and is quite hard to customize the way we want.

File: src/components/ItemCell/index.tsx [hunk 1]
```diff
@@ -234,11 +231,18 @@ export default memo(function ItemCell({
...
-            <Link href={`/my-ottos/${equippedByOtto.id}`} passHref>
+            {!currentOtto && (
+              <Link href={`/my-ottos/${equippedByOtto.id}`} passHref>
+                <StyledOttoLink onClick={e => e.stopPropagation()}>
+                  <Image src={equippedByOtto.image} layout="responsive" width={24} height={24} />
+                </StyledOttoLink>
+              </Link>
+            )}
+            {currentOtto && (
               <StyledOttoLink onClick={e => e.stopPropagation()}>
```
Comment (pyc92) [inline src/components/ItemCell/index.tsx:242]:
```suggestion
              <StyledOttoLink as="span">
```

I this case, it's not a link anymore.

**Review by pyc92 — CHANGES_REQUESTED** [review]

File: src/components/ItemCell/index.tsx (hunk 1 above)
Comment (eonwarped) [reply src/components/ItemCell/index.tsx]: Done

**Comment by eonwarped** [comment]
For referecnce, taken from https://reacttooltip.github.io/react-tooltip/#:~:text=Type%20something...%27%20/%3E%20%0A%3C/ReactTooltip%3E-,Override,-position

**Comment by pyc92** [comment]
@eonwarped 

Could you try using `ReactDOM.createPortal` to solve the issue? I haven't dig into it yet, but I guess the problem stems from where we rendered the tooltip element. Therefore, createPortal should be able to resolve the issue. If it's not possible, then the current solution is ok for now.

**Comment by eonwarped** [comment]
> @eonwarped
> 
> Could you try using `ReactDOM.createPortal` to solve the issue? I haven't dig into it yet, but I guess the problem stems from where we rendered the tooltip element. Therefore, createPortal should be able to resolve the issue. If it's not possible, then the current solution is ok for now.

I actually tried that first, but it didn't work when I put it to #modalRoot , it just disappeared. But it's possible I messed it up or should have used a different element.

**Review by pyc92 — APPROVED** [review]

**Pull request merged**